modules/flowable-cmmn-image-generator/src/main/java/org/flowable/cmmn/image/impl/DefaultCaseDiagramCanvas.java

```diff
@@ -364,7 +366,7 @@ public void drawStage(String name, GraphicInfo graphicInfo, double scaleFactor)
         g.drawRoundRect(x, y, width, height, 6, 6);
 
         // Add the name as text, vertical
-        if (scaleFactor == 1.0 && name != null && name.length() > 0) {
+        if (new BigDecimal(scaleFactor).compareTo(new BigDecimal("1.0"))==0  && name != null && name.length() > 0) {
```

Comment:
@qiudaoke the change to BigDecimal should not be necessary to do a check for a double scalefactor comparison to 1.0. Can you remove the BigDecimal changes from the PR?